spec/lex.dd

```diff
@@ -355,7 +355,12 @@ $(GNAME TokenString):
 $(H3 $(LNAME2 wysiwyg, Wysiwyg Strings))
 
         $(P
-        Wysiwyg ("what you see is what you get") quoted strings are enclosed by `r"` and `"`.
+        Wysiwyg ("what you see is what you get") quoted strings can be defined
```

Comment:
Shouldn't it be all-caps (WYSIWYG) since it's an acronym ?

Reply:
Yes, but lowercase and Titlecase are used elsewhere so this is a style decision that must be made by the documentation's owner or manager. I can just try to fix the English.